data_steward/tools/load_vocab.py

```diff
@@ -0,0 +1,270 @@
+        job_config = LoadJobConfig()
+        job_config.schema = safe_schema
+        job_config.skip_leading_rows = 1
+        job_config.field_delimiter = FIELD_DELIMITER
+        job_config.max_bad_records = MAX_BAD_RECORDS
+        job_config.quote_character = ''
+        source_uri = f'gs://{bucket_name}/{blob.name}'
+        load_job = bq_client.load_table_from_uri(source_uri,
+                                                 destination,
+                                                 job_config=job_config)
+        load_jobs.append(load_job)
+        LOGGER.info(f'table:{destination} job_id:{load_job.job_id}')
+    wait_jobs(load_jobs)
+    return load_jobs
+
+
+def load(project_id,
+         bq_client,
+         src_dataset_id,
+         dst_dataset_id,
+         overwrite_ok=False):
+    """
+    Transform safely loaded tables and store results in target dataset.
+
+    :param project_id:
+    :param bq_client:
+    :param src_dataset_id:
+    :param dst_dataset_id:
+    :param overwrite_ok: if True and the dest dataset already exists the dataset is recreated
+    :return:
+    """
+    if overwrite_ok:
+        bq_client.delete_dataset(dst_dataset_id,
+                                 delete_contents=True,
+                                 not_found_ok=True)
+    bq_client.create_dataset(dst_dataset_id)
+    src_tables = list(bq_client.list_tables(dataset=src_dataset_id))
+
+    job_config = QueryJobConfig()
+    query_jobs = []
+    for src_table in src_tables:
+        schema = bq.get_table_schema(src_table.table_id)
+        destination = f'{project_id}.{dst_dataset_id}.{src_table.table_id}'
+        table = bq_client.create_table(Table(destination, schema=schema),
+                                       exists_ok=True)
+        job_config.destination = table
+        query = SELECT_TPL.render(project_id=project_id,
+                                  dataset_id=src_dataset_id,
+                                  table=src_table.table_id,
+                                  fields=schema)
+        query_job = bq_client.query(query, job_config=job_config)
+        LOGGER.info(f'table:{destination} job_id:{query_job.job_id}')
+        query_jobs.append(query_job)
+    wait_jobs(query_jobs)
+
+
+def main(project_id: str, bucket_name: str, dst_dataset_id: str):
+    """
+    Transform and load vocabulary files from Athena to a BigQuery dataset
+
+    :param project_id:
+    :param bucket_name: refers to the bucket containing vocabulary files from bundle
+    :param dst_dataset_id: final destination to load the vocabulary in BigQuery
+    """
+    bq_client = bq.get_client(project_id=project_id)
+    sandbox_dataset_id = get_sandbox_dataset_id(dst_dataset_id)
+    sandbox_dataset = bq.define_dataset(
+        project_id,
+        sandbox_dataset_id,
+        f'Vocabulary loaded from gs://{bucket_name}',
+        label_or_tag={'type': 'vocabulary'})
+    sandbox_dataset = bq_client.create_dataset(sandbox_dataset, exists_ok=True)
```

Comment:
Can we not just do the following? 

```suggestion
    sandbox_dataset = bq.create_dataset(
        project_id,
        sandbox_dataset_id,
        f'Vocabulary loaded from gs://{bucket_name}',
        label_or_tag={'type': 'vocabulary'},
        overwrite_existing=True)
```

Reply:
I did this; `create_dataset` probably didn't exist when this was written.